doc/graph/fusion_patterns/norm_fusion_patterns.md

```diff
@@ -0,0 +1,64 @@
+Norm Fusion Patterns {#dev_guide_graph_norm_fusion_patterns}
+===========================================================
+
+## Overview
+
+The Norm category for inference includes operations such as:
+GroupNorm, LayerNorm and BatchNormInference.
+
+oneDNN supports various Norm fusion patterns to optimize performance and
+reduce memory bandwidth requirements. This document describes the supported
+fusion patterns for Norm.
+
+## Pattern Structure
+
+oneDNN defines floating-point Norm fusion patterns as follows.
+The blue nodes are required when defining a Norm fusion pattern while the
+brown nodes are optional.
+
+![Norm pattern](images/norm_pattern.png)
+
+1. **Norm Operation**: Performs the corresponding norm operation for the `src`
+   tensor. See the [GroupNorm](@ref dev_guide_op_groupnorm),
+   [LayerNorm](@ref dev_guide_op_layernorm), [BatchNormInference](@ref dev_guide_batch_normalization)
```

Comment:
```suggestion
   [LayerNorm](@ref dev_guide_op_layernorm), [BatchNormInference](@ref dev_guide_op_batchnorminference)
```

Reply:
Fixed comments, thanks for the suggestions!